app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,18 @@
+class LikesController < ApplicationController
+  before_action :require_login
+  
+  def create
+    @user = current_user
+    @image = Image.find(params[:image_id])
+    @like = @image.likes.create
```

Comment:
This will create a like for the image, but the user will not be specified. Setting `@user = current_user` just assigns that variable, but doesn't do anything else. You can get rid of that line and make line 7:

``` rb
@image.likes.create(user: current_user)
```